### PR TITLE
enable debug info by flag in docfx.json

### PR DESCRIPTION
**Pull request by Peachying**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/4774)

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.Engine/PostProcessors/PostProcessorsManager.cs [hunk 1]
```diff
@@ -96,11 +96,7 @@ private static void AddBuildInPostProcessor(List<PostProcessor> processorList)
                     ContractName = "html",
                     Processor = new HtmlPostProcessor
                     {
-                        Handlers =
-                        {
-                            new ValidateBookmark(),
-                            new RemoveDebugInfo(),
-                        },
+
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.Engine/PostProcessors/PostProcessorsManager.cs]: Can simplify this to `Processor = new HtmlPostProcessor()`

File: src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessor.cs [hunk 2]
```diff
@@ -27,6 +27,12 @@ public string GetIncrementalContextHash()
 
         public ImmutableDictionary<string, object> PrepareMetadata(ImmutableDictionary<string, object> metadata)
         {
+            Handlers.Add(new ValidateBookmark());
+            if (!metadata.ContainsKey("_enableDebugInfo") || ((bool)metadata["_enableDebugInfo"]) == false)
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessor.cs]: naming: what about `_keepDebugInfo`?

Comment (Peachying) [reply src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessor.cs]: Ok.

File: src/Microsoft.DocAsCode.Build.Engine/PostProcessors/PostProcessorsManager.cs (hunk 1 above)
Comment (Peachying) [reply src/Microsoft.DocAsCode.Build.Engine/PostProcessors/PostProcessorsManager.cs]: Got it~

**Pull request merged**